extensions/arc/src/models/postgresModel.ts

```diff
@@ -196,6 +178,63 @@ export class PostgresModel extends ResourceModel {
 		}
 	}
 
+	private async createCoordinatorEngineSettings(provider: azdata.QueryProvider, ownerUri: string, skip: String[]): Promise<void> {
+		const engineSettingsCoordinator = await provider.runQueryAndReturn(ownerUri, 'select name, setting, short_desc,min_val, max_val, enumvals, vartype from pg_settings');
+
+		this.coordinatorNodeEngineSettings = [];
+		engineSettingsCoordinator.rows.forEach(row => {
+			let rowValues = row.map(c => c.displayValue);
+			let name = rowValues.shift();
+			if (!skip.includes(name!)) {
+				let result: EngineSettingsModel = {
+					parameterName: name,
+					value: rowValues.shift(),
+					description: rowValues.shift(),
+					min: rowValues.shift(),
+					max: rowValues.shift(),
+					options: rowValues.shift(),
+					type: rowValues.shift()
+				};
+
+				this.coordinatorNodeEngineSettings.push(result);
+			}
+		});
+
+	}
+
+	private async createWorkerEngineSettings(provider: azdata.QueryProvider, ownerUri: string, skip: String[]): Promise<void> {
+
+		let querySuccessCheck = await provider.runQueryAndReturn(ownerUri, `select count(result) from run_command_on_workers('select json_agg(pg_settings) from pg_settings') where success;`);
+		if (querySuccessCheck.rows[0][0].displayValue === '0') {
+			querySuccessCheck = await provider.runQueryAndReturn(ownerUri, `select result from run_command_on_workers('select json_agg(pg_settings) from pg_settings') limit 1;`);
+			throw new Error(querySuccessCheck.rows[0][0].displayValue);
+		}
+
+		const engineSettingsWorker = await provider.runQueryAndReturn(ownerUri, `select result from run_command_on_workers('select json_agg(pg_settings) from pg_settings') where success limit 1;`);
```

Comment:
I think this needs to be done in 1 query rather than 3.  Both for speed/efficiency, and because the success check isn't guaranteed to work.  The success check might succeed, but the real query could fail.  Or the success check could fail, and then the real query could work.

If any worker returns successfully, we want to return a single row to reduce the size of the payload.  (Don't need results from all <n> workers since we expect their configuration to be the same).

But if all workers return an error, I think it'd be useful to capture error messages from all of them rather than a single one.  (They may have failed for different reasons).

I'd suggest something like this:

```
with settings as (select nodename, success, result from run_command_on_workers('select json_agg(pg_settings) from pg_settings') order by success desc)
select * from settings limit case when exists(select 1 from settings where success) then 1 end
```

Sorting by success and the contitional limit means we'll take a single successful row if there are any.  Otherwise we return all the failed rows.